db/migrate/20210202044303_rename_contributors.rb

```diff
@@ -0,0 +1,5 @@
+class RenameContributors < ActiveRecord::Migration[6.1]
+  def change
+    rename_table 'contributors', 'abstract_contributors'
```

Comment:
It seems the contributors are not so abstract anymore. Worth re-considering the name?

Reply:
They absolutely are abstract, we never instantiate on of them.

Reply:
The `abstract_class = true` is really poorly named. That is one strategy for mapping the inheritance structure of the classes to tables, the pure STI strategy is another.